sdk/storage/azblob/bloberror/error_codes.go

```diff
@@ -149,3 +149,7 @@ const (
 	UnsupportedQueryParameter                         Code = "UnsupportedQueryParameter"
 	UnsupportedXMLNode                                Code = "UnsupportedXmlNode"
 )
+
+var (
+	SharedKeyDoesNotExist = errors.New("SAS can only be signed with a SharedKeyCredential")
```

Comment:
What about `NoSharedKeyCredential`, or `MissingSharedKeyCredential` (I think I like this one better)?  I think having the term `SharedKeyCredential` is important as it ties it to the name of the type.